api/endpoints/news.py

```
@@ -16,58 +19,76 @@ async def get_news_sources():
         return {"sources": sources}
     except Exception as e:
         logger.error(f"Error getting news sources: {str(e)}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)